content/doc/pipeline/index.adoc

```diff
@@ -1,981 +1,437 @@
+image::/images/pipeline/jenkins-workflow.png[title="Jenkins Workflow", 800]
 
-# Creating a Pipeline
+== Pipeline Defined
 
-Now that you have Jenkins running and have installed the Pipeline plugin, you are ready to create your first pipeline. Create a new pipeline by selecting New Item from the Jenkins home page.
+Pipelines are Jenkins jobs enabled by the Pipeline (formerly called “workflow”) plugin and built with simple text scripts that use a Pipeline DSL (domain-specific language) based on the Groovy programming language. 
 
-First, give your pipeline a name, eg: "hello-world-flow".  pipelines are simple, Groovy scripts, so let's add the obligatory Hello World. Add a pipeline to the Pipeline script textarea:
+Pipelines leverage the power of multiple steps to execute both simple and complex tasks according to parameters that you establish. Once created, pipelines can build code and orchestrate the work required to drive applications from commit to delivery.
 
-```groovy
-echo 'Hello world'
-```
+== Pipeline Vocabulary
 
-Now save your pipeline, ensuring the Use Groovy Sandbox option is checked (more details to follow on this setting). Click Build Now to run your pipeline.
+Pipeline terms such as “step,” “node,” and “stage” are a subset of the vocabulary used for Jenkins in general. 
 
-## Editing your Pipeline
+Step::
+    A “step” (often called a “build step”) is a single task that is part of sequence. Steps tell Jenkins what to do.
 
-Because pipelines are simple text scripts, they are easy to edit. As you've seen, pipelines can be edited directly in the Jenkins UI when configuring your pipeline.
+Node::
+    In pipeline coding contexts, as opposed to Jenkins generally, a “node” is a step that does two things, typically by enlisting help from available executors on agents:
+    * Schedules the steps contained within it to run by adding them to the Jenkins build queue (so that as soon as an executor slot is free on a node, the appropriate steps run). 
+    * Creates a workspace, meaning a file directory specific to a particular job, where resource-intensive processing can occur without negatively impacting your pipeline performance. Workspaces last for the duration of the tasks assigned to them.
 
-![Pipeline Editor](images/2-workflow-edit-hr.png)
 
-## Using the Snippet Generator
-To make editing your pipelines easier, use the Snippet Generator. The Snippet Generator is dynamically populated with the latest Pipeline steps. Depending on the plugins installed in your environment, you may see more available steps.
+(In Jenkins generally, a “node” means any computer that is part of your Jenkins installation, whether that computer is used as a master or as an agent).
 
-![Snippet Generator](images/3-snippet-generator-hr.png)
+Stage::
+    A “stage” is a step that calls supported APIs. Pipeline syntax is comprised of stages. Each stage can have one or more build steps within it. 
```

Comment:
> A "stage" is a step that calls supported APIs.

This doesn't make sense to me/doesn't tell me anything. Really a `stage` is a step that helps denote different parts of the execution. It's important to me that we communicate `node`, which has a Closure containing other steps, and `stage`, which is kind of a marker in a procedural sequence of steps, differently